CMakeLists.txt

```diff
@@ -6,32 +6,36 @@ set(CMAKE_BUILD_TYPE Release)
 #set(CMAKE_CXX_COMPILER /usr/local/bin/g++-4.8)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/misc/")
-find_package(LibXml2 REQUIRED)
```

Comment:
so I used the exiv2 build server libraries and avoided needing to link libxml2 again..

http://exiv2.dyndns.org:8080/userContent/builds/Categorized/Platform/msvc/